src/Domain/CoordinateMaps/TimeDependent/Translation.cpp

```diff
@@ -69,16 +116,38 @@ std::optional<std::array<double, Dim>> Translation<Dim>::inverse(
              << function_of_time.size()
              << ") does not match the dimension of the translation map (" << Dim
              << ").");
+  double radial_function_value = 1.0;
+  if (f_of_r_ != nullptr) {
```

Comment:
This function can similarly be optimized a bit. Let me know if you want to discuss this one as well.